libraries/AC_PID/AC_PID.cpp

```diff
@@ -252,9 +252,9 @@ float AC_PID::update_all(float target, float measurement, float dt, bool limit,
     P_out *= _pid_info.Dmod;
     D_out *= _pid_info.Dmod;
 
-    // boost output if required
-    P_out *= boost;
-    D_out *= boost;
+    // scale output if required
```

Comment:
If this is a user exposed parameter should we consider adding protection against negative values or 0.